plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiOutputExtension.kt

```diff
@@ -92,15 +92,27 @@ class JvmAbiOutputExtension(
                             private val keptMethods = mutableListOf<MethodNode>()
                             private val innerClassInfos = mutableMapOf<String, InnerClassInfo>()
 
+                            override fun visit(
+                                version: Int,
+                                access: Int,
+                                name: String?,
+                                signature: String?,
+                                superName: String?,
+                                interfaces: Array<out String>?,
+                            ) {
+                                val prunedAccess = if (prune) access or Opcodes.ACC_PUBLIC else access
```

Comment:
Same as in `@Metadata`. Not sure if I should fallback to package-private here instead.